lib/awful/screen.lua

```diff
@@ -503,9 +503,8 @@ function screen.object.get_tiled_clients(s, stacked)
     -- Remove floating clients
     for _, c in pairs(clients) do
         if not c.floating
-            and not c.fullscreen
```

Comment:
why fullscreen was removed?
or `immobilized_*` is set when fullscreen is on?

Reply:
Yup: https://github.com/awesomeWM/awesome/blob/1518b0f4c0e533b9cf076f0b1f418c99e4d20747/lib/awful/client.lua#L802-L803